modules/videopress/class.videopress-player.php

```diff
@@ -146,10 +146,10 @@ public function asXML() {
 		if ( empty( $this->video ) || is_wp_error( $this->video ) )
 			return '';
 
-		if ( isset( $this->options['freedom'] ) && $this->options['freedom'] === true )
-			$content = $this->html5_static();
-		else
+		if ( isset( $this->options['force_flash'] ) && $this->options['force_flash'] === true )
```

Comment:
Since you're touching that code, it might be worth switching to [yoda conditions](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#yoda-conditions) and [braces](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#brace-style).